### PR TITLE
Revert "add profile detail page"

### PR DESCRIPTION
**Pull request by JunZheng007**
Reverts hatchways/team-wintergreen#60

### CODE REVIEW

**Review by rajivtitus — APPROVED** [review]

File: client/src/components/RequestForm/RequestForm.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  for (let i = hour; i < 24; i++) {
+    times.push(`${i % 12 ? i % 12 : 12} ${i / 12 < 1 ? 'am' : 'pm'}`);
+  }
+  return times;
+};
+
+const RequestForm = ({ className, profile, handleSubmit }: Props): JSX.Element => {
+  const classes = useStyles();
+  const today = new Date();
+  console.log(today.toDateString());
```
Comment (ezeilo-su) [inline client/src/components/RequestForm/RequestForm.tsx:50]: Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!

File: client/src/components/RequestForm/RequestForm.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                            ? props.values.dropInDate.getHours()
+                            : 0,
+                        )}
+                        renderInput={(params) => <TextField {...params} />}
+                        sx={{ width: '40%' }}
+                      />
+                    </Box>
+                  </Box>
+                  <Box display="flex" flexDirection="column" marginTop={3}>
+                    <label className={classes.label}>Drop Off</label>
```
Comment (ezeilo-su) [inline client/src/components/RequestForm/RequestForm.tsx:100]: Use MUI `InputLabel`

File: client/src/components/RequestForm/RequestForm.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                    <Box display="flex">
+                      <DatePicker
+                        onChange={(value) => props.setFieldValue('dropOffDate', value)}
+                        value={props.values.dropOffDate}
+                        renderInput={(params) => <TextField sx={{ width: '60%' }} {...params} />}
+                        // sx={{ width: '50%' }}
```
Comment (ezeilo-su) [inline client/src/components/RequestForm/RequestForm.tsx:106]: Let's get rid of this comment

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
Hey @JunZheng007, good progress on this one. Kindly address those comments below.
Happy coding!

File: client/src/components/RequestForm/RequestForm.tsx (hunk 1 above)
Comment (JunZheng007) [reply client/src/components/RequestForm/RequestForm.tsx:50]: fix it on PR #70

File: client/src/components/RequestForm/RequestForm.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                  'dropInDate',
+                  (dropInDate, schema) => dropInDate && schema.min(dropInDate),
+                ),
+              })}
+              onSubmit={handleSubmit}
+            >
+              {(props) => (
+                <Form className={classes.form}>
+                  <Box display="flex" flexDirection="column">
+                    <label className={classes.label}>Drop in</label>
```
[the remark this answers is not in the document]
Comment (JunZheng007) [reply client/src/components/RequestForm/RequestForm.tsx:79]: solved it on PR #70

**Comment by JunZheng007** [comment]
all changes merged to PR #70

**Pull request closed without merging**